op-node/flags/p2p_flags.go

```diff
@@ -353,5 +354,13 @@ func P2PFlags(envPrefix string) []cli.Flag {
 			Required: false,
 			EnvVars:  p2pEnv(envPrefix, "SYNC_REQ_RESP"),
 		},
+		&cli.BoolFlag{
+			Name:     P2PPingingName,
+			Usage:    "Enables P2P ping-pong background service",
+			Value:    true, // on by default
+			Hidden:   true, // hidden, only here to disable in case of bugs.
+			Required: false,
+			EnvVars:  p2pEnv(envPrefix, "PINGING"),
```

Comment:
nit category & this is not a deprecated p2p flag b/c I'd like to remove all of the deprecated P2 flags.

Reply:
We argued last week that new features should have flags to disable them. This allows exactly that. We do want it to be on by default though, and it's hidden because we want to phase out the flag once we have confidence it doesn't have to be disabled.

Reply:
I agree that a flag for this is good. Maybe we want to add it to a feature flag category rather than a deprecated category. Deprecated IMO means flags that don't do anything anymore or are actively going to be removed. I'm not sure the timeline for removing this flag, but it seems like this flag will live for a while.

Reply:
So do we want to add it to a global `p2p` category? The other p2p definitions don't have a category either. And can make `Hidden: false` if you prefer it.

Reply:
`hidden: true` is totally fine. I thought that most of the other P2P flags have the category